packages/ihe-gateway/server/CodeTemplates/Common Library/File type/File type.js

```diff
@@ -86,7 +86,7 @@ function isLikelyTextFile(fileBuffer) {
 }
 
 function isLikelyXML(contents) {
-  if (contents && typeof contents === "string") {
+  if (contents) {
```

Comment:
This was changed on [another PR](https://github.com/metriport/metriport/pull/1752). Is this PR pointing to the right branch?

Reply:
1696 now points to 1752

Reply:
[The comment on that PR](https://github.com/metriport/metriport/pull/1752#discussion_r1520107295) was not answered/addressed